tests/test_checker/test_noqa.py

```diff
@@ -61,6 +61,7 @@ def test_noqa_fixture_disabled(absolute_path, all_violations):
         'Z222': 1,
         'Z223': 1,
         'Z224': 1,
+        'Z225': 0,
```

Comment:
It is not really a complexity violation. I would say that this is a consistency issue.